web-components/src/components/button/Button.ts

```diff
@@ -337,7 +330,7 @@ export namespace Button {
             @keydown=${(e: KeyboardEvent) => this.handleKeyDown(e)}
             role=${this.role}
             tabindex=${ifDefined(this.tabIndex || undefined)}
-            aria-pressed=${this.ariaPressed === 'true' ? true : false}
+            aria-pressed=${this.ariaPressed === "true" ? true : false}
```

Comment:
why no aria-disable added for tag = "a". Also better to add examples under sandbox for (a and input) tag with disabled state.